apps/cli/src/commands/environment/list.environment.ts

```diff
@@ -1,5 +1,5 @@
 import BaseCommand from '../base.command'
```

Comment:
@rajdip-b  Can you verify if this is correct cause I was getting error in this file?